### ISSUE TITLE
Console output - add date/time and output all to a log file

### ISSUE THREAD

**Issue by daddywookie** [21jake/Binance-volatility-trading-bot-JS#8]
When there is a long period where nothing happens it can be hard to keep track of the executions. It might be helpful to add a timestamp to the output to make it clearer what is happening and when.

```
[2021/05/18 18:05:02] No coin has risen more than 5% in the last 5 minutes
[2021/05/18 18:05:02] MATICUSDT price has hit SL threshold
[2021/05/18 18:05:03] The asset MATICUSDT has been sold sucessfully at the profit of -5.72% and recorded in sold-assets.json
[2021/05/18 18:05:03] Wait for the bot to gather data to check price volatility...
```
Also, outputting the console to a .log file would help with troubleshooting. This might need to be configured with some form of log rotation.

**Comment by 21jake** [comment]
Fair point, implemented since 21jake@2d9fb75

**Comment by daddywookie** [comment]
Will update and test on my next run.
